src/map/map-includes-keys/map-includes-keys.predicate.ts

```diff
@@ -0,0 +1,14 @@
+import { isMap } from '../../type/is-map'
+
+/**
+ * @category Predicates
+ * @param value The value to validate.
+ * @param required List of keys that can be set for this set.
+ * @typeParam Key The type of keys to check for.
+ */
+export function mapIncludesKeys<Key = unknown>(
+    value: unknown,
+    required: Iterable<Key>
+): value is Map<unknown, unknown> {
+    return isMap(value) && [...value.keys()].every(key => [...required].includes(key as Key))
```

Comment:
This only checks that all keys in `required` are present in the given map. It does _not_check if `required` has any additional keys.

```typescript
@MapIncludesKeys(["A", "B", "X"])
public map = new Map(["A", "B"])
```

This would fail, which is not what I would expect the validator to do (regardless of whether its called `MapIncludesKeys` or `MapContainsKeysExactly`.